sixtyfps_compiler/tests/syntax/analysis/binding_loop_layout2.60

```diff
@@ -1,6 +1,30 @@
 // Copyright © SixtyFPS GmbH <info@sixtyfps.io>
 // SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
 
+Wrap := Rectangle {
+    property woo <=> text.wrap;
+
+    VerticalLayout {
+//  ^error{The binding for the property 'layout-cache' is part of a binding loop}
+//  ^^error{The binding for the property 'height' is part of a binding loop}
+        HorizontalLayout {
+//      ^error{The binding for the property 'layout-cache' is part of a binding loop}
+//      ^^error{The binding for the property 'width' is part of a binding loop}
+//      ^^^error{The binding for the property 'layoutinfo-v' is part of a binding loop}
+            text := Text {
+                text: "Hello World";
+            }
+            square := Rectangle {
+//                   ^error{The binding for the property 'height' is part of a binding loop}
+                width: height;
+//                    ^error{The binding for the property 'width' is part of a binding loop}
+                background: violet;
+            }
+        }
+        Rectangle {}
+    }
+    property <bool> test: square.width == square.height;
```

Comment:
Is this unused?

Reply:
`test`  is read by the interpreter test, and it would panic if there was a real loop

Reply:
True, but I think the interpreter instantiates the `Test` below, not `Wrap`. Maybe that needs a "forwarding"?